pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>org.jenkins-ci.test</groupId>
     <artifactId>docker-fixtures</artifactId>
-    <version>1.3-SNAPSHOT</version>
+    <version>1.4-SNAPSHOT-fbelzunc-udp-ports</version>
```

Comment:
No do not change the `version` in a PR. Just `mvn -DskipTests clean deploy` your branch.